Dockerfile

```diff
@@ -42,17 +41,10 @@ RUN source scl_source enable rh-python36 && \
     fix-permissions /opt/app-root && \
     rpm-file-permissions
 
-COPY ./.s2i/bin/ /usr/libexec/s2i
-
-ENV APPROOT="/opt/app-root/src/tensorflowapp-sample"  VERSION="1"
-COPY ["tensorflowapp-sample", "${APPROOT}"]
-COPY ["requirements.txt", "${APPROOT}"]
+COPY ./.s2i/bin/usage /tmp/scripts/
 
-WORKDIR $APPROOT
+RUN ls -la /tmp/scripts
 
+COPY ["requirements.txt", "/opt/app-root/src/"]
+RUN pip install --upgrade pip
 RUN pip install -r requirements.txt --default-timeout=100
-
-USER 1001
-
-ENTRYPOINT [ "/usr/libexec/s2i/uid_entrypoint" ]
-CMD ["/usr/libexec/s2i/usage"]
```

Comment:
Again this was here as a best practice/model code, so removing it makes it harder for a newbie to add it and this sample useful for something real

Reply:
Changing this to `CMD ["/tmp/scripts/usage"]` as I have removed `COPY ./.s2i/bin/ /usr/libexec/s2i`
The scripts are copied into `/tmp/scripts` on its own during `make build`. Removing redundancy.